rs/protobuf/def/registry/subnet/v1/subnet.proto

```diff
@@ -92,12 +92,12 @@ message SubnetRecord {
 }
 
 message EcdsaInitialization {
-  registry.crypto.v1.EcdsaKeyId key_id = 1;
```

Comment:
Nit: Looks like the `registry.crypto` import is no longer needed in this file. Maybe elsewhere too.

Reply:
Indeed: given that there are no more hits when searching for "registry.crypto" in this file, one would think that the import can be removed. I just tried that, but interestingly then the protobuf-generator fails. This is because the import is used twice in the file via `crypto.v1.AlgorithmId`. I tried and it also worked with `registry.crypto.v1.AlgorithmId`!? I'll leave it as-is for now, unless you prefer otherwise.

In `metadata.proto` and `idkg.proto`, the import can be removed, however, so I just did that.